src/components/manage/Blocks/Description/Edit.jsx

```diff
@@ -50,6 +50,16 @@ class Edit extends Component {
     onAddBlock: PropTypes.func.isRequired,
     onFocusPreviousBlock: PropTypes.func.isRequired,
     onFocusNextBlock: PropTypes.func.isRequired,
+    editable: PropTypes.bool,
```

Comment:
This means that we have to migrate all our existing blocks, to add this prop?

There's already support for readOnly blocks, if you pass data.readOnly. Can we hitchike on that?

@avoinea Is it correct what I'm saying?

Reply:
no, you don't have to migrate your blocks. For volto blocks, default value for editable is 'true'. 
For readonly blocks implemented by @avoinea i can only see the readOnlyBehavior field in ContentTypeLayout, but i can't understand how it works.. maybe @avoinea can you show us?

Reply:
Maybe then the wording is not appropriate? I had the same confusion as @tiberiuichim did. @giuliaghisini what is the ultimate purpose of this prop?

Reply:
This props enable user to select the block and view it's configuration in the sidebar, to compare or replicate it in the translation content that it's editing in that moment, but without editing the block.

Reply:
However, question: If a block does not implement a mean for being "readonly" it could be modified, right? and if so, we will not only to change all the existing blocks, but also transfer that burden to the block code itself. 

Can't we make it depend on the Engine?

Reply:
yes.. i don't think it's easy to make it dependent from engine, because the editable param passed to block has to be passed to the Editor text component to make it disabled..

Reply:
For consistency I would use `props.readOnly` also. I don't think we can use `data.readOnly` for this purpose as this is persistent to ZODB.